brain/compartmentReportReader.h

```diff
@@ -0,0 +1,243 @@
+/* Copyright (c) 2017, EPFL/Blue Brain Project
```

Comment:
-2 :)
The classes are small and pretty related to each other. I suggest to take a look at the header file and decide later. My opinion is that having multiple files of 4 lines is not needed here.